glue_iam.tf

```diff
@@ -111,22 +111,8 @@ data "aws_iam_policy_document" "glue_policy_document" {
     actions = [
       "glue:*",
     ]
-    effect = "Allow"
-    resources = flatten([
-      [format("arn:%s:glue:%s:%s:catalog",
-        data.aws_partition.current.partition,
-        data.aws_region.current.name,
-      data.aws_caller_identity.current.account_id)],
-      aws_glue_catalog_database.shepherd[*].arn,
-      [for bucket in var.subscriber_buckets : [
-        format("arn:%s:glue:%s:%s:table/%s/*",
```

Comment:
We probably can do this as `"arn:%s:glue:%s:%s:table/*/*",` and just not list buckets.